entity_id.go

```diff
@@ -94,17 +100,15 @@ func _CheckChecksum(ledgerID string, address string) string {
 	asciiA := []rune("a")[0]
 	w := 31
 
-	id := ledgerID + "000000000000"
+	zeros, _ := hex.DecodeString("000000000000")
```

Comment:
There is no point in decoding a string of 0's. If you know the value you can just hard code it. In fact you might be able to create a `const` for this.

Reply:
Wait i didn't see this comment yesterday some reason, but yes its fixed.